cni-plugin/pkg/plugin/plugin.go

```diff
@@ -109,7 +110,7 @@ func cmdAdd(args *skel.CmdArgs) (err error) {
 	// a proper error to the runtime.
 	defer func() {
 		if e := recover(); e != nil {
-			msg := fmt.Sprintf("Calico CNI panicked during ADD: %s", e)
+			msg := fmt.Sprintf("Calico CNI panicked during ADD: %s\nStack: %s", e, string(debug.Stack()))
 			if err != nil {
 				// If we're recovering and there was also an error, then we need to
 				// present both.
```

Comment:
In the test - there was not log from line 122 "Final result of ..." 

So I can only assume the CI is not displaying the logs from the plugin - might be worth adding that in somehow.  Not sure what is involved there.  If we had that then it would be sufficient I guess to just add the stack trace to that log.

Reply:
The CI should be displaying logs from the plugin... I did a chunk of work recently to ensure that it was (and was doing so only for failed tests). But maybe I missed something....